### PR TITLE
Onboarding page

### PR DESCRIPTION
**Pull request by Denis-Dinkov**

### CODE REVIEW

File: src/views/onboarding/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            Have any ideas about Example? Request example here.
+          </button>
+        </div>
+        <div
+          className="px-16"
+          data-title="Custom"
+        >
+          <NotFound />
+        </div>
+      </Tabs >
```
Comment (pivanov) [inline src/views/onboarding/index.tsx:120]: ` >` 🤣

File: src/views/onboarding/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      <RequestExampleModal onClose={toggleVisibility} />
+    </div>
+  );
+};
+
+export default Onboarding;
```
Comment (pivanov) [inline src/views/onboarding/index.tsx:126]: yeah ... here the default export is correct ... because this one is used un routes with lazy import ;)

**Review by pivanov — CHANGES_REQUESTED** [review]

File: src/views/onboarding/components/githubExamples/index.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { NotFound } from '../notFound';
+
+export const GithubExamples = () => {
+  return (
+    <div>
```
Comment (pivanov) [inline src/views/onboarding/components/githubExamples/index.tsx:5]: we don't need this extra div here ... and we don't need GithubExamples component at all if we use it only to add this extra div here

File: src/views/onboarding/components/defaultExamples/index.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    setSearchQuery,
+  ] = useState('');
+
+  const filteredSnippets = snippets.filter((snippet) =>
+    snippet.name.toLowerCase().includes(searchQuery.toLowerCase()),
+  );
+
+  const handleSearch = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(e.target.value);
+  }, []);
```
Comment (pivanov) [inline src/views/onboarding/components/defaultExamples/index.tsx:29]: What do you think?

```suggestion
const [
    filteredSnippets,
    setFilteredSnippets,
  ] = useState(snippets);

  const handleSearch = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
    const query = e.target.value;
    const filtered = snippets.filter((snippet) =>
      snippet.name.toLowerCase().includes(query.toLowerCase()),
    );
    setFilteredSnippets(filtered);
  }, []);
  ```

Comment (Denis-Dinkov) [reply src/views/onboarding/components/defaultExamples/index.tsx:29]: Nice one :)

**Review by pivanov — APPROVED** [review]
r+

**Pull request merged**